.github/CODEOWNERS

```diff
@@ -83,9 +83,9 @@ platforms/core-execution/hashing/                           @gradle/bt-execution
 platforms/core-execution/snapshots/                         @gradle/bt-execution @gradle/bt-dv-build-cache
 
 # Develocity integration
-platforms/enterprise/                                   @gradle/bt-build-scan @gradle/dv-build-insights-team
-platforms/enterprise/enterprise/                        @gradle/bt-build-scan @gradle/dv-testing-team @gradle/dv-build-insights-team @ldaley
-platforms/enterprise/enterprise-logging/                @gradle/bt-build-scan @gradle/dv-build-insights-team @gradle/dv-testing-team
+platforms/enterprise/                                   @gradle/bt-build-scan @gradle/dv-integrations-team
+platforms/enterprise/enterprise/                        @gradle/bt-build-scan @gradle/dv-testing-team @gradle/dv-integrations-team @ldaley
+platforms/enterprise/enterprise-logging/                @gradle/bt-build-scan @gradle/dv-integrations-team @gradle/dv-testing-team
```

Comment:
@gradle/dv-integrations-team Are you still interested in being code owners of these directories? If not, let's remove you.

Reply:
I think you can remove us. Thanks!